recipe/meta.yaml

```
@@ -55,10 +55,12 @@ outputs:
         - pytorch {{ pytorch }} {{ torch_proc_type }}*
```

probably want the same in host?
```suggestion
        - pytorch
```

---

Um, i guess we should add the torch_proc_type in the host too. the thing is, we *need* the GPU stuff (as far as I know).

---

I added a comment in the yaml as to why we can have two constrains in the building sections, but not in the running sections.

---

Fine by me. 👍